README.adoc

```diff
@@ -16,31 +16,10 @@ This repository uses https://github.com/golang/go/wiki/Modules[Go modules]. You
 
 == Development
 
-To run this operator locally you need to have at least one Minishift profile started:
+toolchain-e2e repo makefile can be used as a starting point for setting up host and member operators for development.
```

Comment:
I think it would be helpful to provide a link as well.
```suggestion
The dev targets in the toolchain-e2e repository can be used to build and deploy the host and member operators for development: https://github.com/codeready-toolchain/toolchain-e2e/blob/master/make/dev.mk
```